### ISSUE TITLE
Turoborepo causing PreviewJS to throw errors and/or no compatible plugin for workspace.

### ISSUE THREAD

**Issue by Zendrex** [fwouts/previewjs#1490]
### Describe the bug

Hey there, I'm attempting to get the VSC extension to work my with monorepo (via turborepo), however it looks like the extension, as well as the CLI are unable to navigate to the required node_modules. This causes the extension to entirely fail.

### Reproduction

Create a turborepo.
Create a nuxt3 app.
Create a basic component library (based on https://github.com/wuruoyun/vue-component-lib-starter)
Try to use the PreviewJS extension.

### Preview.js version

1.19.6

### Framework

Turbo latest, Nuxt 3.3.1, Vue 3.2.25

### System Info

```shell
System:
  OS: Windows 10 10.0.22621
  CPU: (32) x64 AMD Ryzen 9 5950X 16-Core Processor
  Memory: 47.42 GB / 63.91 GB
Binaries:
  Node: 18.15.0 - C:\Program Files\nodejs\node.EXE
  Yarn: 3.5.0 - ~\AppData\Roaming\npm\yarn.CMD
  npm: 9.5.0 - C:\Program Files\nodejs\npm.CMD
IDEs:
  VSCode: 1.76.2 - E:\Microsoft VS Code\bin\code.CMD
Browsers:
  Edge: Spartan (44.22621.1413.0), Chromium (111.0.1661.51)
  Internet Explorer: 11.0.22621.1
```

### Used Package Manager

yarn

### Extension logs (useful for crashes)

```shell
Unable to read installed version of package: nuxt Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in e:\***\monorepo\node_modules\nuxt\package.json
    at new NodeError (node:internal/errors:399:5)
    at exportsNotFound (node:internal/modules/esm/resolve:361:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:641:13)
    at resolveExports (node:internal/modules/cjs/loader:565:36)
    at Module._findPath (node:internal/modules/cjs/loader:634:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1061:27)
    at Function.resolve (node:internal/modules/cjs/helpers:116:19)
    at Object.readInstalledVersion (file:///C:/***/***/.vscode/extensions/zenclabs.previewjs-1.19.6/dist/node_modules/.pnpm/@previewjs+core@19.0.2_@types+node@18.15.5/node_modules/@previewjs/core/dist/index.mjs:1317:47)
    at Object.isCompatible (file:///C:/***/***/.vscode/extensions/zenclabs.previewjs-1.19.6/dist/node_modules/.pnpm/@previewjs+plugin-vue3@8.0.4_h6rrpcx33yr4dcgxzju5ogtziu/node_modules/@previewjs/plugin-vue3/dist/index.mjs:490:102)
    at async Module.setupFrameworkPlugin (file:///C:/***/***/.vscode/extensions/zenclabs.previewjs-1.19.6/dist/node_modules/.pnpm/@previewjs+core@19.0.2_@types+node@18.15.5/node_modules/@previewjs/core/dist/index.mjs:1282:9) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

No compatible Preview.js plugin for workspace: e:\***\***\monorepo\apps\website
No workspace root detected from e:\***\***\monorepo\packages\components\package.json
```

### Repo link (if available)

Inapplicable, but can provide a boilerplate if needed.

### Anything else?

package.json (root)
```json
{
	"name": "my-turborepo",
	"version": "0.0.0",
	"private": true,
	"workspaces": [
		"apps/*",
		"packages/*"
	],
	"scripts": {
		"build": "turbo run build",
		"dev": "turbo run dev",
		"lint": "turbo run lint",
		"format": "prettier --write \"**/*.{ts,tsx,md}\""
	},
	"dependencies": {
		"@types/node": "^18.15.7"
	},
	"devDependencies": {
		"prettier": "latest",
		"rimraf": "^4.4.1",
		"turbo": "latest",
	},
	"engines": {
		"node": ">=16.9.0"
	},
	"packageManager": "yarn@3.5.0",
}
```

package.json (packages/components)
```json
{
	"name": "@zendrex/components",
	"version": "0.0.0",
	"scripts": {
		"build": "rimraf dist && vue-tsc && vite build",
		"docs:dev": "vitepress dev docs",
		"docs:build": "vitepress build docs",
		"docs:serve": "vitepress serve docs"
	},
	"types": "./dist/types/index.d.ts",
	"module": "./dist/my-lib.es.js",
	"exports": {
		".": {
			"import": "./dist/my-lib.es.js"
		},
		"./dist/style.css": "./dist/style.css"
	},
	"devDependencies": {
		"@types/node": "^18.15.7",
		"@vitejs/plugin-vue": "^4.1.0",
		"rimraf": "^4.4.1",
		"sass": "^1.60.0",
		"typescript": "^5.0.2",
		"vite": "^4.2.1",
		"vitepress": "^1.0.0-alpha.61",
		"vue": "^3.2.47",
		"vue-tsc": "^1.2.0"
	},
	"peerDependencies": {
		"vue": "^3.2.25"
	},
	"packageManager": "yarn@3.5.0"
}
```

package.json (apps/website)
```json
{
	"private": true,
	"scripts": {
		"build": "nuxt build",
		"dev": "nuxt dev",
		"generate": "nuxt generate",
		"preview": "nuxt preview",
		"postinstall": "nuxt prepare"
	},
	"dependencies": {
		"@zendrex/components": "*"
	},
	"devDependencies": {
		"nuxt": "^3.3.1"
	},
}
```

.yarnrc.yml
```yml
nodeLinker: node-modules

plugins:
  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
    spec: "@yarnpkg/plugin-interactive-tools"
  - path: .yarn/plugins/@yarnpkg/plugin-typescript.cjs
    spec: "@yarnpkg/plugin-typescript"
  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
    spec: "@yarnpkg/plugin-workspace-tools"

yarnPath: .yarn/releases/yarn-3.5.0.cjs

```

**Comment by fwouts** [comment]
Thanks @Zendrex for the bug report!

I suspect this is not due to turborepo (in fact the Preview.js codebase itself uses it) but your logs are really helpful.

Will try to fix for the next release.

**Comment by fwouts** [comment]
Hey @Zendrex, the fix is now live in v1.20!

Let me know how you go, feel free to reopen the issue if it's not addressed.
